src/Users/UserCredentialsEdit.js

```diff
@@ -1,113 +1,99 @@
-import React, { Component } from "react";
-import { connect } from "react-redux";
-import { Link, Redirect } from "react-router-dom";
-import { updateUserCredential } from "../store/auth";
+import React, {Component} from "react";
+import { connect } from 'react-redux';
+import { Link, Redirect } from 'react-router-dom';
+import { updateUserCredential } from '../store/auth';
 
 class UserCredentialsEdit extends Component {
-  constructor() {
-    super();
-    this.state = {
-      username: "",
-      password: "",
-      passwordConfirm: "",
-    };
-    this.update = this.update.bind(this);
-    this.validatePassword = this.validatePassword.bind(this);
-  }
-
-  async update(e) {
-    //       e.preventDefault();
-    console.log("This State");
-    console.log(this.state);
-    const information = {
-      username: this.state.username,
-      password: this.state.password,
-      passwordConfirm: this.state.passwordConfirm,
-      isGuest: false,
-    };
-    console.log(`Generate data from component`);
-    console.log(information);
-    try {
-      console.log(`Transfer data from component`);
-      console.log(information);
-      await this.props.update(information);
-    } catch (ex) {
-      console.log(ex);
+    
+    constructor(){
+        super();
+        this.state = {
+            username:'',
+            password:'',
+            passwordConfirm:''
+        }
+        this.update = this.update.bind(this);
+         this.validatePassword = this.validatePassword.bind(this);
     }
-  }
-
-  //   componentDidMount(){
-  // this.setState({ username:this.props.auth.username, password:this.props.auth.password, passwordConfirm:this.props.auth.password  });
-  //   }
-  validatePassword() {
-    return (
-      this.state.password.length > 0 &&
-      this.state.passwordConfirm.length > 0 &&
-      this.state.password === this.state.passwordConfirm
-    );
-  }
+    
+    async update(e){
+ //       e.preventDefault();
+        const information = {username:this.state.username, password:this.state.password, passwordConfirm:this.state.passwordConfirm };
```

Comment:
```suggestion
        const information = {username:this.state.username, password:this.state.password, passwordConfirm:this.state.passwordConfirm
        isGuest: false};
```
can you please add this back it not affect your request unless the person is a guest